src/app/gen-queries.ts

```diff
@@ -438,14 +438,6 @@ export const SampleQueries: SampleQuery[] = [
     "docLink": "https://developer.microsoft.com/en-us/graph/docs/api-reference/v1.0/api/worksheet_list_tables",
     "tip": "This query requires a driveItem id.  To find the ID of the driveItem that corresponds to an Excel Workbook, you can run: GET https://graph.microsoft.com/v1.0/me/drive/root/search(q='.xlsx')?select=name,id,webUrl."
 },
-{
```

Comment:
Would it be easier to comment this out for now? We will have to add this back in when the sample is fixed. 

Maybe in the future it would be nice to have an "enabled/disabled" attribute.

Reply:
It will be added back the next time I import the samples. Commenting it out would have the same effect. I'm hoping that it will be fixed by then.